### DISCUSSION TITLE
Welcome to rvc-monitor-py Discussions!

### DISCUSSION THREAD

**Discussion by linuxkidd** [linuxkidd/rvc-monitor-py#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rlust** [comment]
Happy to help with testing! I have an Entegra Aspire 2022 and using the CP version from ERVIN on pi 3 with CAN, and running latest HomeAssistant on pi4. I am using the HUE integration on HA to control lights and awnings and Node-red in Home Assistant to read sensors with MQTT. 

Able to develop Node-red flows and test with Home Assistant.
